zebra/zebra_nb_config.c

```diff
@@ -1252,7 +1261,8 @@ int lib_vrf_zebra_l3vni_id_modify(struct nb_cb_modify_args *args)
 						    pfx_only ? 1 : 0, 1)
 		    != 0) {
 			if (IS_ZEBRA_DEBUG_VXLAN)
-				zlog_debug(
+				snprintf(
```

Comment:
why the change to snprintf?

Reply:
User needs to be informed for any validation failure(s), errmsg buffer used by northbound framework to relay to user. 
I have removed debug filter as this vrf-vni mapping failure user need to be notified. 
The errmsg description can be displayed via vtsyh, grpc, etc.